e2e/test_get_immunization.py

```diff
@@ -36,3 +36,18 @@ def get_deleted_imms(self):
         imms = self.create_a_deleted_immunization_resource(self.default_imms_api)
         response = self.default_imms_api.get_immunization_by_id(imms["id"])
         self.assert_operation_outcome(response, 404)
+
+    def test_get_imms_with_tbc_pk(self):
+        """it should get a FHIR Immunization resource if the nhs number is TBC"""
+        imms = create_an_imms_obj()
+        del imms["contained"][1]["identifier"][0]["value"]
+        imms["contained"][1]["identifier"][0]["extension"][0]["valueCodeableConcept"]["coding"][0]["code"] = "04"
+
+        response = self.default_imms_api.create_immunization(imms)
```

Comment:
Please use `self.create_immunization_resource(self.default_imms_api, imms)` it has error handling.

Reply:
done